.github/workflows/tox.yml

```diff
@@ -14,9 +14,11 @@ jobs:
           - '3.9'
           - '3.10'
           - '3.11'
+          - '3.12'
         pip-version:
-          - 22.0.4
-          - 23.0.1
+          # - 22.0.4
+          # - 23.0.1
+          - 23.2.1
```

Comment:
This makes me think that we should probably replace this matrix with something else. It doesn't make a ton of sense to run the latest Python with an age-old `pip`, just like it doesn't make sense to run an ancient Python with bleeding-edge `pip`.

Reply:
A proposed fix for this is in https://github.com/mrtmm/hastexo-xblock/pull/8.

Reply:
Ack, thanks, I pulled that in and tests are passing :)